ndohyep/settings/base.py

```diff
@@ -216,3 +220,21 @@
 TEMPLATE_DIRS = (
     os.path.join(PROJECT_ROOT, 'ndohyep'),
 )
+
+
+# Google Analytics
+
+GOOGLE_ANALYTICS = {
+    'google_analytics_id': '',
+}
+GOOGLE_ANALYTICS_IGNORE_PATH = ['', ]
```

Comment:
Let's make sure we add `'/health/'` here and the `google_analytics_id` is something we set at deploy time with puppet.